packages/eslint-config-norton/rules/typescript-base.js

```diff
@@ -1,5 +1,14 @@
 module.exports = {
 	rules: {
+		/**
+		 * Avoid export default to improve TypeScript experience. Applicable in
```

Comment:
One last change: you can remove this entire comment and replace it with one that just references the issue in this repo now that we've discussed it (`https://github.com/wwnorton/style/issues/70`).